.github/workflows/cd-deploy-nodes-gcp.yml

```diff
@@ -334,7 +252,7 @@ jobs:
   # Note: this instances are not automatically replaced or deleted
   deploy-instance:
     name: Deploy single ${{ inputs.network }} instance
-    needs: [ build, test-configuration-file ]
+    needs: [ build ]
```

Comment:
This should depend on the config file tests, because we don't want to deploy nodes if the image fails.